Resources/ConfigPresets/DeltaV/deltav.toml

```diff
@@ -53,3 +58,21 @@ enabled = false # only enabled on periapsis
 eligible_number = 5
 initiator_ghost_requirement = false
 voter_ghost_requirement = false
+
+[atmos]
+space_wind = false # janky as fuck and probably not suitable for mrp
```

Comment:
also pro this was supposed to be true :trollface: 
space wind fun for everyone involved, why do you think we have sec magboots